app-backup/amanda/amanda-3.5.4.ebuild

```diff
@@ -0,0 +1,524 @@
+	"${FILESDIR}"/${PN}-3.5.4-no-ndmp.patch
+)
+
+MYFILESDIR="${T}/files"
+ENVDIR="/etc/env.d"
+ENVDFILE="97amanda"
+TMPENVFILE="${T}/${ENVDFILE}"
+
+# This is a complete list of Amanda settings that the ebuild takes from the
+# build environment. This allows users to alter the behavior of the package as
+# upstream intended, but keeping with Gentoo style. We store a copy of them in
+# /etc/env.d/97amanda during the install, so that they are preserved for future
+# installed. This variable name must not start with AMANDA_, as we do not want
+# it captured into the env file.
+ENV_SETTINGS_AMANDA="
+AMANDA_GROUP_NAME
+AMANDA_USER_NAME AMANDA_USER_HOMEDIR
+AMANDA_SERVER AMANDA_SERVER_TAPE AMANDA_SERVER_TAPE_DEVICE AMANDA_SERVER_INDEX
+AMANDA_TAR_LISTDIR AMANDA_TAR
+AMANDA_PORTS_UDP AMANDA_PORTS_TCP AMANDA_PORTS_BOTH AMANDA_PORTS
+AMANDA_CONFIG_NAME AMANDA_TMPDIR"
+
+amanda_variable_setup() {
+	# Setting vars
+	local currentamanda
+
+	# Grab the current settings
+	currentamanda="$(set | grep -E "^AMANDA_" | grep -v '^AMANDA_ENV_SETTINGS' | xargs)"
+
+	# First we set the defaults
+	AMANDA_GROUP_NAME=amanda
+	AMANDA_USER_NAME=amanda
+	AMANDA_USER_HOMEDIR=/var/spool/amanda
+
+	# This installs Amanda, with the server. However, it could be a client,
+	# just specify an alternate server name in AMANDA_SERVER.
+	: ${AMANDA_SERVER:=${HOSTNAME}}
+	: ${AMANDA_SERVER_TAPE:=${AMANDA_SERVER}}
+	: ${AMANDA_SERVER_TAPE_DEVICE:="${EPREFIX}"/dev/nst0}
+	: ${AMANDA_SERVER_INDEX:=${AMANDA_SERVER}}
+	: ${AMANDA_TAR_LISTDIR:=${AMANDA_USER_HOMEDIR}/tar-lists}
+	: ${AMANDA_CONFIG_NAME:=DailySet1}
+	: ${AMANDA_TMPDIR:="${EPREFIX}"/var/tmp/amanda}
+	: ${AMANDA_DBGDIR:=${AMANDA_TMPDIR}}
+	# These are left empty by default
+	: ${AMANDA_PORTS_UDP:=""}
+	: ${AMANDA_PORTS_TCP:=""}
+	: ${AMANDA_PORTS_BOTH:=""}
+	: ${AMANDA_PORTS:=""}
+
+	# What tar to use
+	: ${AMANDA_TAR:=/bin/tar}
+
+	# Now pull in the old stuff
+	if [[ -f ${EROOT}${ENVDIR}/${ENVDFILE} ]]; then
+		# We don't just source it as we don't want everything in there.
+		eval $(grep -E "^AMANDA_" "${EROOT}${ENVDIR}/${ENVDFILE}" | grep -v '^AMANDA_ENV_SETTINGS')
+	fi
+
+	# Re-apply the new settings if any
+	[[ -n ${currentamanda} ]] && eval $(echo "${currentamanda}")
+}
+
+pkg_setup() {
+	amanda_variable_setup
+
+	# If USE=minimal, give out a warning, if AMANDA_SERVER is not set to
+	# another host than HOSTNAME.
+	if use minimal && [[ ${AMANDA_SERVER} == ${HOSTNAME} ]]; then
+		elog "You are installing a client-only version of Amanda."
+		elog "You should set the variable \$AMANDA_SERVER to point at your"
+		elog "Amanda-tape-server, otherwise you will have to specify its name"
+		elog "when using amrecover on the client."
+		elog "For example: Use something like"
+		elog "AMANDA_SERVER=\"myserver\" emerge amanda"
+		elog
+	fi
+}
+
+src_unpack() {
+	# we do not want the perl src_unpack
+	default
+}
+
+src_prepare() {
+	default
+	use ndmp && eapply "${FILESDIR}"/${P}-tirpc.patch
```

Comment:
```
 * Applying amanda-3.5.4-tirpc.patch ...
/var/tmp/portage/app-backup/amanda-3.5.4/temp/environment: line 825: /var/tmp/portage/app-backup/amanda-3.5.4/files/amanda-3.5.4-tirpc.patch: No such file or directory
/var/tmp/portage/app-backup/amanda-3.5.4/temp/environment: line 828: /var/tmp/portage/app-backup/amanda-3.5.4/files/amanda-3.5.4-tirpc.patch: No such file or directory
 [ !! ]
 * ERROR: app-backup/amanda-3.5.4::gentoo failed (prepare phase):
 *   patch -p1  failed with /var/tmp/portage/app-backup/amanda-3.5.4/files/amanda-3.5.4-tirpc.patch
 *
```

Reply:
Doesn't appear to be needed with *3.5.4* as a `PKG_CHECK_MODULES` for `libtirpc` was added. Tested compilation both with and without `USE=ndmp` and verified that `libndmlib-3.5.4.so` depends on `libtirpc.so.3`.